src/pages/Cart/cart.css

```diff
@@ -0,0 +1,129 @@
+.cart-cont {
+  display: flex;
+  gap: 2rem;
+  padding: 2rem 1rem;
+  justify-content: center;
+}
+.cart-left-cont {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+.cart-right-cont {
+  width: 340px;
```

Comment:
Instead of px using rem  will help in mobile responsive too

Reply:
I need it to be fixed.